guides/common/modules/con_projectserver-overview.adoc

```diff
@@ -3,18 +3,26 @@
 [id="{ProjectServerID}-Overview_{context}"]
 = {ProjectServer} overview
 
-{ProjectServer} is the central component of a {Project} deployment where you plan and manage the content lifecycle.
+{ProjectServer} is the central component of a {Project} deployment.
 
 A typical {Project} deployment includes one {ProjectServer} on which you perform the following operations:
 
+ifndef::foreman-el,foreman-deb[]
 * Content lifecycle management
+endif::[]
 * Configuration of {SmartProxyServers}
 * Configuration of hosts
 * Host provisioning
+ifndef::foreman-el,foreman-deb[]
 * Patch management
 * Subscription management
+endif::[]
 
-{ProjectServer} delegates content distribution, host provisioning, and communication to {SmartProxyServers}.
+{ProjectServer} delegates
+ifndef::foreman-el,foreman-deb[]
+content distribution,
+endif::[]
+host provisioning, and communication to {SmartProxyServers}.
```

Comment:
Right now, the comma after "host provisioning" is wrong for foreman-el and foreman-deb. I suggest to simply duplicate the lines and use two macros.